content/v1/getting-started/installation/installationwizard/src/static/js/tests/generate-yaml.test.js

```diff
@@ -891,7 +891,7 @@ describe("GIVEN createYamlString function", () => {
       version: v2.12.0
       images:
         driverRepository: dellemc
```

Comment:
Is the value for driver repo correct?

Reply:
Retrieved it from powerstore-> values.yaml

Reply:
it is constant used in testfile, updated it